libs/pika/async_mpi/tests/unit/mpi_ring_async_sender_receiver.cpp

```diff
@@ -188,12 +184,12 @@ int pika_main(pika::program_options::variables_map& vm)
             }
         }
 
-        auto tt = mpi::detail::get_num_requests_in_flight();
-        if (tt != 0)
-        {
-            std::cout << "Rank " << rank << " flight " << tt << " counter "
-                      << counter << std::endl;
-        }
+        //        auto tt = mpi::detail::get_num_requests_in_flight();
+        //        if (tt != 0)
+        //        {
+        //            std::cout << "Rank " << rank << " flight " << tt << " counter "
+        //                      << counter << std::endl;
+        //        }
```

Comment:
Debugging?

Reply:
yup. forgot about that. Frankly I don't mind commented out debug stuff in tests as it helps people later when they try to finx some new bug they discover

Reply:
Guard it with this instead of commenting it out: https://github.com/pika-org/pika/blob/cd738d191967f7f8656954372b93d8cc6e7997fd/libs/pika/async_mpi/tests/unit/mpi_ring_async_sender_receiver.cpp#L235?